ibis/pyspark/compiler.py

```diff
@@ -992,6 +992,20 @@ def compile_window_op(t, expr, scope, **kwargs):
     ]
 
     pyspark_window = Window.partitionBy(grouping_keys).orderBy(ordering_keys)
+
+    if not isinstance(operand.op(), ops.ShiftBase):
```

Comment:
What happened for ShiftBase?

Reply:
I feel like should be some kind of exception. If the user use ShiftBase with a unbounded window, it almost doesn't make sense. We should throw some kind of exception here.

Reply:
For shift base, Spark sets the window bounds and throws an exception when different bounds are set. The expected behavior for ibis is to not throw an exception. Adding a comment here to explain why we don't set bounds for ShiftBase operations.